src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -19,6 +19,7 @@
 FLOW_TOOLS_JSON_GEN_TIMEOUT = 60
 PROMPT_FLOW_DIR_NAME = ".promptflow"
 HOME_PROMPT_FLOW_DIR = (Path.home() / PROMPT_FLOW_DIR_NAME).resolve()
+SERVICE_CONFIG_FILE = "pf.yaml"
```

Comment:
we should already have a constant for this? as this is sharing with authoring scenario?

Reply:
It's used to be hard code in configuration, replaced to constant